### DISCUSSION TITLE
Is there any way to conditional shown/hide some of navigation links

### DISCUSSION THREAD

**Discussion by samg666** [Budibase/budibase#3662] [Help] [answered]
Hello,

Some of my pages was accessible for ADMIN only, but it still shown on the navigation bar. Is there any way to conditional show/hide particular links? 

![image](https://user-images.githubusercontent.com/29649651/144569702-35dfe286-84c6-4f85-be4d-874252e9709c.png)

**Comment by mjashanks** [comment, marked as answer]
I was about to answer yes, but have found an issue. https://github.com/Budibase/budibase/issues/3665

This issue lists the steps that you should take - but is currently broken. We will look at this asap.

**Comment by samg666** [comment]
Thanks @mjashanks , it works for me on version 0.9.173. The issue of #3665 seems to be a bug on new version.
